content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,87 @@
+
+The in-proxy generation of service-level metrics is a drop in replacement for the HTTP metrics currently generated by Mixer, namely: `istio_requests_total`, `istio_request_duration_*` and `istio_request_size`.
+
+## Enable Service-level Metrics Generation in Envoy
+
+To enable the generation of service-level metrics in Envoy, execute the following steps:
+
+1. Disable calls to `istio-telemetry` in the mesh
+
+    In order to prevent duplicate telemetry generation, turn off the use of the `istio-telemetry` service for telemetry generation.
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set mixer.telemetry.enabled=false --set mixer.policy.enabled=false
+    {{< /text >}}
+
+    Alternatively, comment out `mixerCheckServer` and `mixerReportServer` from the mesh config (the `istio` config map).
+
+1. Enable the metadata exchange filter
+
+    For the generation of service-level metrics, proxies must exchange workload metadata. This exchange is handled by a custom filter.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://github.com/istio/proxy/blob/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
+    {{< /text >}}
+
+1. Enable the custom stats filter
+
+    A custom filter must also be applied to actually generate the service-level metrics.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/stats_filter.yaml
+    {{< /text >}}
+
+1. See the results
+
+    Visit Grafana "Istio Mesh" dashboard. It will display the same telemetry as before -- without any requests flowing through Istio's Mixer.
+
+## Differences
+
+There are some small differences in the generation of the service-level metrics at the moment. Full feature-parity is a requirement before this
+functionality can be considered stable. In the meantime, please be aware of these differences:
+
+1. The `istio_request_duration_*` metric uses more granular buckets inside the proxy. This results in lower latency measurements in the histograms than before.
+1. The new metric is called `istio_request_duration_milliseconds` (instead of `istio_request_duration_seconds`). The Grafana dashboard is updated to account for the new metric.
+
+## Performance impact
+
+{{< warning >}}
+
+The focus of the initial release of this functionality has not been on performance optimization. We have several planned performance-related improvements
+in the pipeline at this time.
+
+This feature will not be considered for promotion to Beta or Stable status until our performance goals have been realized.
+
+{{< /warning >}}
+
+1. The `istio-telemetry` deployment can be switched off. This should save ~0.5 vCPU per 1000 rps of mesh traffic. This also halves the CPU consumed by Istio while collecting Istio standard metrics.
+1. The new filters use 10% less CPU for `istio-proxy` containers than the Mixer filter.
```

Comment:
The new filters `together`    , otherwise it is unclear if it is 10% less each or combined.

Reply:
done.